libs/shareProcessor.js

```diff
@@ -85,6 +85,7 @@ module.exports = function(logger, poolConfig){
             redisCommands.push(['rename', coin + ':shares:roundCurrent', coin + ':shares:round' + shareData.height]);
             redisCommands.push(['rename', coin + ':shares:timesCurrent', coin + ':shares:times' + shareData.height]);
             redisCommands.push(['sadd', coin + ':blocksPending', [shareData.blockHash, shareData.txHash, shareData.height, shareData.worker, dateNow].join(':')]);
+            redisCommands.push(['hset', 'Allblocks', coin + "-" + shareData.height, [shareData.blockHash, shareData.txHash, shareData.height, shareData.worker, dateNow].join(':')]); //used for block stat
```

Comment:
I think there is a better way to implement `this.getBlocksStats` api call without double saving the block data in redis.